### PR TITLE
Pullup4to5

### PR DESCRIPTION
**Pull request by willy-ahva**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by tseho — APPROVED** [review]

File: src/Akeneo/Channel/Bundle/Resources/translations/validators.fi_FI.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pim_enrich.entity.locale:
```
Comment (anaisbaunelemaire) [inline src/Akeneo/Channel/Bundle/Resources/translations/validators.fi_FI.yml:1]: Only en_US translation files should be merged :-)

File: package.json [hunk 2]
```diff
@@ -111,6 +111,7 @@
   "dependencies": {
     "ajv": "^6.10.2",
     "akeneo-design-system": "link:akeneo-design-system",
+    "@types/backbone": "1.4.8",
```
Comment (jmleroux) [inline package.json:114]: I think we can remove this one.

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/product/form/categories.js [hunk 3]
```diff
@@ -131,7 +131,11 @@ define([
             this.treeAssociate.lock(false);
           }
 
-          this.delegateEvents();
+                    if (this.isReadOnly()) {
```
Comment (nmarniesse) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/product/form/categories.js:134]: duplcated if. And indent is weird

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/translations/jsmessages.fi_FI.yml [hunk 4]
```diff
@@ -0,0 +1,154 @@
+pim_menu.item:
```
Comment (anaisbaunelemaire) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/translations/jsmessages.fi_FI.yml:1]: Same here ;-)

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/translations/messages.fi_FI.yml [hunk 5]
```diff
@@ -0,0 +1,2 @@
+akeneo_connectivity.connection:
```
Comment (anaisbaunelemaire) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/translations/messages.fi_FI.yml:1]: Same

**Review by jmleroux — COMMENTED** [review]
Lock files should be regenerated.

And languages files could be ignored

**Comment by willy-ahva** [comment]
warmup

**Review by jmleroux — APPROVED** [review]

**Pull request merged**